apps/editors/tests/test_views.py

```diff
@@ -438,13 +456,79 @@ def test_new_editors(self):
         anchors = doc('#editors-stats .editor-stats-table:eq(2)').find('td a')
         eq_(anchors.eq(0).text(), self.user.display_name)
 
+    def test_unlisted_queues_only_for_senior_reviewers(self):
+        listed_queues_links = [
+            reverse('editors.queue_fast_track'),
+            reverse('editors.queue_nominated'),
+            reverse('editors.queue_pending'),
+            reverse('editors.queue_prelim'),
+            reverse('editors.queue_moderated')]
+
+        # Only listed queues for editors.
+        doc = pq(self.client.get(self.url).content)
+        queues = doc('#listed-queues ul li a')
+        queues_links = [link.attrib['href'] for link in queues]
+        assert queues_links == listed_queues_links
+        assert not doc('#unlisted-queues')  # Unlisted queues are not visible.
+
+        # Both listed and unlisted queues for senior editors.
+        self.login_as_senior_editor()
+        doc = pq(self.client.get(self.url).content)
+        queues = doc('#listed-queues ul li a')
+        queues_links = [link.attrib['href'] for link in queues]
+        assert queues_links == listed_queues_links
+        unlisted_queues = doc('#unlisted-queues ul li a')
+        unlisted_queues_links = [link.attrib['href']
+                                 for link in unlisted_queues]
+        assert unlisted_queues_links == [
```

Comment:
It's a bit weird that `listed_queues_links` is a list of static URLs while `unlisted_queues_links` is a list of URLs from the page. Can you make them both lists of static URLs at the top of the functions so the two tests follow the same pattern?